include/mbedtls/mbedtls_config.h

```diff
@@ -564,12 +564,13 @@
 /**
  * \def MBEDTLS_AES_FEWER_TABLES
  *
- * Use less ROM/RAM for AES tables.
+ * Use less ROM/RAM for AES tables and code.
  *
  * Uncommenting this macro omits 75% of the AES tables from
  * ROM / RAM (depending on the value of \c MBEDTLS_AES_ROM_TABLES)
  * by computing their values on the fly during operations
- * (the tables are entry-wise rotations of one another).
+ * (the tables are entry-wise rotations of one another). It
+ * also enables some additional code-size optimisations.
```

Comment:
I think it's clear in the original form of this paragraph that things would slow down (because of "computing their values on the fly during operations") but I think we should be explicit in the added sentence:

```
It also enables some additional code-size optimisations, , again at
the expense of performance over code size
```